### PR TITLE
add: home page

### PR DESCRIPTION
**Pull request by harshitpaliwal95**
- add: home page
Screenshot 2022-02-19 at 11 23 35 PM

- add: category page
Screenshot 2022-02-19 at 10 30 22 PM

-add: rules page
Screenshot 2022-02-19 at 11 24 27 PM

### CODE REVIEW

**Comment by TJain02** [comment]
looks good!

**Review by TJain02 — APPROVED** [review]

**Review by TJain02 — APPROVED** [review]

**Comment by ashifkhn** [comment]
This looks amazing.

**Comment by ArpitPatni** [comment]
Looks good to merge

**Review by bmendhe23 — APPROVED** [review]
Looks good to merge.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+                src="assets/luffy.jpg"
+                alt="loading"
+                srcset=""
+              />
+            </div>
+            <div class="card-badge">new</div>
+            <div class="card-comp-contant">
+              <h2 class="card-h2">Monkey D Luffy</h2>
+              <h3 class="card-h3">Take this quizz to test yourself!</h3>
+              <h3 class="card-h3">5 Question</h3>
```
Comment (agrawalshanu708) [inline index.html]: try to avoid names in class which has semantic meaning like h2,h3

**Pull request merged**